forge-game/src/main/java/forge/game/phase/PhaseHandler.java

```diff
@@ -264,20 +264,83 @@ private boolean isSkippingPhase(final PhaseType phase) {
         }
     }
 
+    public boolean givePriorityToPlayer() {
+        if (isSkippingPhase(phase)) {
+            return true;
```

Comment:
why would this return true when it gets skipped? 😕

Reply:
Good catch, it should be false.

Reply:
why resolve, still seems wrong...?